_delphi_utils_python/delphi_utils/runner.py

```diff
@@ -1,8 +1,12 @@
 """Indicator running utilities."""
+import argparse as ap
+import importlib
 from typing import Any, Callable, Dict, Optional
 from .archive import ArchiveDiffer
+from .archive import from_params as archiver_from_params
```

Comment:
why not name each function `archiver/validator_from_params` in the definition? is it so there's a common function name across each of the objects?

Reply:
Done